frontend/src/pages/mocap/MocapTrialModal.tsx

```diff
@@ -382,6 +382,20 @@ const MocapTrialModal = observer((props: MocapTrialModalProps) => {
                             }
                         };
                     }
+                } else {
+                    if (scales[""] == null) {
```

Comment:
Maybe if(!("" in scales) || !("type" in scales[""])) or something? The "== null" always makes me nervous

Reply:
I agree, I am going to change it in all of the if cases.

Reply:
I changed it to the method hasOwnProperty